controllers/provider.go

```diff
@@ -331,3 +335,55 @@ func (c *ApiController) TestProviderConnection() {
 	}
 	c.ResponseOk()
 }
+
+func (c *ApiController) validateProviderURLs(provider object.Provider) {
```

Comment:
move to map where key - field, value - error 
than iterate over map and call IsURLValid

Reply:
done